pkg/orchestrator/assetscanestimationwatcher/watcher.go

```diff
@@ -78,8 +78,8 @@ func (w *Watcher) GetAssetScanEstimations(ctx context.Context) ([]AssetScanEstim
 	logger := log.GetLoggerFromContextOrDiscard(ctx)
 	logger.Debugf("Fetching AssetScanEstimations which need to be reconciled")
 
-	filter := fmt.Sprintf("state/state ne '%s' and state/state ne '%s'",
-		models.AssetScanEstimationStateStateDone, models.AssetScanEstimationStateStateFailed)
+	filter := fmt.Sprintf("(state/state ne '%s' and state/state ne '%s') or (deleteAfter eq null or '%s' > deleteAfter)",
```

Comment:
When comparing time, you should use `gt` and not `>`.
Please look at the following for example
```
(lastSeen eq null or lastSeen lt %s)", discoveryTime.Format(time.RFC3339)))
```

Reply:
Thanks, wasn't sure about it, wanted to check this logic but the VMClarity images was broken